### PR TITLE
40: Added .name to author entry to get name to display.

### PR DESCRIPTION
**Pull request by echicoine-icf**
https://github.com/cqframework/sample-content-ig/issues/40

### CODE REVIEW

**Comment by echicoine-icf** [comment]
image

File: templates/liquid/Measure.liquid [hunk 1]
```diff
@@ -127,7 +127,7 @@
         {% if Measure.author.exists() %}
```
Comment (brynrhodes) [inline templates/liquid/Measure.liquid:127]: If we're changing what we're displaying, we should also change what we're conditioning the display on, this should probably now be Measure.author.name.exists() yes?

File: templates/liquid/Measure.liquid [hunk 2]
```diff
@@ -127,7 +127,7 @@
...
         <tr>
             <th scope="row"><b>Author: </b></th>
-            <td style="padding-left: 4px;">{{Measure.author}}</td>
+            <td style="padding-left: 4px;">{{Measure.author.name}}</td>
         </tr>
         {% endif %}
```
Comment (brynrhodes) [inline templates/liquid/Measure.liquid:133]: This looks good for Measure.author, but we should treat all of the ContactDetail-valued elements the same way then, update the templates for Measure.editor, Measure.reviewer, and Measure.endorser to follow suit

Comment (brynrhodes) [reply templates/liquid/Measure.liquid:133]: Also note that these same elements appear in the other templates (Library, PlanDefinition, and ActivityDefinition) we should update those to be consistent as well

**Review by brynrhodes — CHANGES_REQUESTED** [review]

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**